blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -90,12 +90,16 @@
     SHARD_PUSH_PERIOD("2000"),
     SHARD_PULL_PERIOD("20000"),
 
+    // blueflood uses zookeeper to acquire locks before working on shards
+    ZOOKEEPER_CLUSTER("127.0.0.1:22181"),
```

Comment:
Would it be too much trouble to default this to NONE here? The reason is... existing blueflood consumers (who are not using ZK) would need to know to deploy a config change that explicitly sets this to NONE, when they deploy this code change.